content/cpp/concepts/deque/terms/push_back/push_back.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.push_back()'
+Description: 'Adds an element to the end of the deque.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Containers'
+  - 'OOP'
+  - 'Classes'
+  - 'Deques'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`deque.push_back()`** method adds an element to the end of the `deque`.
+
+## Syntax
+
+The `push_back()` method is called on a `deque` using the following syntax:
+
+```pseudo
+dequeName.push_back(value);
+```
+
+> **Note:** Each `value` in `dequeName` must be of the same [data type](https://www.codecademy.com/resources/docs/cpp/data-types).
+
```

Comment:
Also add an ## Example block here, which will have an example showcasing the term followed by an output block ```shell```